spec/lib/vanagon/engine/pooler_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'vanagon/engine/pooler'
 
 describe 'Vanagon::Engine::Pooler' do
-  let (:platform) { double(Vanagon::Platform) }
+  let (:platform) { double(Vanagon::Platform, :target_user => 'root') }
```

Comment:
It looks like target_user is defaulting to root in platform.rb, does it work without this?